airflow/api_connexion/schemas/dag_run_schema.py

```diff
@@ -72,5 +72,25 @@ class DAGRunCollectionSchema(Schema):
     total_entries = fields.Int()
 
 
+class DagRunsBatchFormSchema(Schema):
+    """ Schema to validate and deserialize the Form(request payload) submitted to DagRun Batch endpoint"""
+
+    class Meta:
+        """ Meta """
+        datetimeformat = 'iso'
+        strict = True
+
+    page_offset = fields.Int(required=False, missing=0, min=0)
+    page_limit = fields.Int(required=False, missing=100, min=1)
+    dag_ids = fields.List(fields.Str(), required=False, missing=None)
+    execution_date_gte = fields.DateTime(required=False, missing=None)
```

Comment:
```suggestion
    execution_date_gte = fields.DateTime()
```
I suggest you remove the missing and required field.  To only use them when the value should be something other than False and None respectively. This is already the default from marshmallow. 
https://marshmallow.readthedocs.io/en/2.x-line/api_reference.html#module-marshmallow.fields

Reply:
Thanks @ephraimbuddy . I have removed `required=False` from the args. But without `missing=None`, the fields were missing in deserialized data.